docs/release-process.md

```diff
@@ -8,73 +24,38 @@
  2. Package two days before a release. So if the release is to be on Thursday,
     package on Tuesday. Think about timezones as well.
 
- 3. Ensure that the tests on Travis CI are green.
+ 3. Ensure that the relevant tests on CI are green.
 
-    See: https://travis-ci.org/php/php-src/builds
+    See: 
+      https://travis-ci.org/php/php-src/builds
+      https://dev.azure.com/phpazuredevops/PHP/
 
     It is recommended to do so a couple of days before the packaging day, to
     have enough time to investigate failures, communicate with the authors and
     commit the fixes.
 
-    The RM for the branch is also responsible for keeping the CI green on
-    ongoing basis between the releases. Check the CI status for your branch
-    periodically and resolve the failures ASAP. See more in
-    https://wiki.php.net/rfc/travis_ci.
+    Check the CI status for your branch periodically and resolve the failures ASAP. 
+    
+    See more in https://wiki.php.net/rfc/travis_ci.
 
- 4. Ensure that Windows builds will work before packaging.
```

Comment:
Why drop this?

Reply:
Now that we have AV mentioned in the CI section under notes, it should be given the same attention as nix, so don't think we need special case it.

The point was to remove the special case, windows is as much our problem as any other OS, on an equal kind of footing.